src/containerapp/azext_containerapp/commands.py

```diff
@@ -50,45 +50,45 @@ def load_command_table(self, _):
     with self.command_group('containerapp env dapr-component') as g:
         g.custom_command('init', 'init_dapr_components', is_preview=True)
 
-    with self.command_group('containerapp service', deprecate_info=self.deprecate(redirect='containerapp add-on', expiration='2.56.1', hide=True), is_preview=True) as g:
+    with self.command_group('containerapp service', deprecate_info=self.deprecate(redirect='containerapp add-on', expiration='2.57.0', hide=True), is_preview=True) as g:
         g.custom_command('list', 'list_all_services')
 
     with self.command_group('containerapp add-on', is_preview=True) as g:
         g.custom_command('list', 'list_all_services')
 
-    with self.command_group('containerapp service redis', deprecate_info=self.deprecate(redirect='containerapp add-on redis', expiration='2.56.1', hide=True)) as g:
+    with self.command_group('containerapp service redis', deprecate_info=self.deprecate(redirect='containerapp add-on redis', expiration='2.57.0', hide=True)) as g:
         g.custom_command('create', 'create_redis_service', supports_no_wait=True)
         g.custom_command('delete', 'delete_redis_service', confirmation=True, supports_no_wait=True)
 
     with self.command_group('containerapp add-on redis') as g:
         g.custom_command('create', 'create_redis_service', supports_no_wait=True)
         g.custom_command('delete', 'delete_redis_service', confirmation=True, supports_no_wait=True)
 
-    with self.command_group('containerapp service postgres', deprecate_info=self.deprecate(redirect='containerapp add-on postgres', expiration='2.56.1', hide=True)) as g:
+    with self.command_group('containerapp service postgres', deprecate_info=self.deprecate(redirect='containerapp add-on postgres', expiration='2.57.0', hide=True)) as g:
```

Comment:
Which version of azure-cli is the expectation of last version that can uses `az containerapp service`?
As long as the expectation is that 2.56.0 is the last usable version, then there is no problem filling in 2.56.1.

Reply:
gotcha. it was a lil up in the air over what specific version we should make the cutoff...the very first cli release or the release right after our add-ons are added. since all add-ons can make it in 2.56.1, then i'll revert the cut-off to 2.56.1. thanks!